constants.h

```diff
@@ -2,7 +2,8 @@
 #define __CONSTANTS_H__
 
 /* Temperature threshold that has to be exceed long enough to start defrosting. */
-const float TEMPERATURE_DELTA_TO_DEFROST = 4.0;
+/* NOT USED */
+/* const float TEMPERATURE_DELTA_TO_DEFROST = 4.0; */
```

Comment:
Was this really what you wanted to do?
Or should this be disabled?
const float TEMPERATURE_DELTA_TO_FORCE_DEFROST = 10.0;

Reply:
Oops, commented out wrong const! Pushed fix already.